File: Backend/Services/Service.HR/Models/WardModel.cs
using Core.CommonModel;
using System.ComponentModel.DataAnnotations;

namespace Service.HR.Models
{
    public class WardModel : BaseModel
    {
        public int Id { get; set; }

        [MaxLength(100)]
        public string Name { get; set; }

        public int DistrictId { get; set; }

        public string DistrictName { get; set; }

        public int ProvinceId { get; set; }

        public string ProvinceName { get; set; }
    }
}

File: FontEnd/erp/src/app/modules/human-resources/ward/ward.model.ts
import { BaseViewModel } from 'src/app/core/models/base.mode';
import { ProvinceViewModel } from '../province/province.model';
import { DistrictViewModel } from '../district/district.model';

export interface WardViewModel extends BaseViewModel {
  name: string;
  districtId: number;
  districtName: string;
  provinceId: number;
  provinceName: string;
}
